src/openforms/emails/digest.py

```diff
@@ -289,7 +297,7 @@ def collect_invalid_certificates() -> list[InvalidCertificate]:
             invalid_certs.append(
                 InvalidCertificate(
                     id=cert.id,
-                    label=cert.label,
+                    label=cert.__str__(),
```

Comment:
```suggestion
                    label=str(cert),
```
you should never call magic methods directly :)

I thought we provided the model instance directly to the template, and then `{{ certificate }}` would have the same effect, but this works too and is the least amount of code changes.